core/app/graphql/maeku/types/entry_content_type.rb

```diff
@@ -0,0 +1,5 @@
+module Maeku
+  class Types::EntryContentType < Types::BaseType
```

Comment:
Super nit-picky but I'd almost just call this ContentType, unless there is a specific reason to call this EntryContent instead.

Reply:
the class it refers to is `Maeku::EntryContent`, which is an object that is always created and associated with a `Maeku::Entry`. That's all.